server.go

```diff
@@ -246,6 +246,8 @@ func runMain() error {
 	html.Get("/logout", application.Route(controller, "Logout"))
 
 	app.Handle("/api/*", api)
+	app.Handle("/assets/*", static)
```

Comment:
Adding the `/assets` endpoint to the `app` router will associate all of the middleware with that endpoint.

See how I used a `parent` router with no middleware to solve this problem

Reply:
Did I get it this time?